SU-SWS/saa_victoria_subtheme
# ADAPT-476: Adjusted banner h2 to be bigger.

# READY FOR REVIEW

# Summary
- Make h2 in banner bigger

# Review By (Date)
- 7/20

# Urgency
- important!

# Steps to Test

1. pull down
1. clear cache
2. check that h2 in banner is bigger - using modular typogrpahy 3 now instead of 2.

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
